exchange/exchangetest/firstpartydata-multibidder-config-valid.json

```diff
@@ -0,0 +1,227 @@
+{
```

Comment:
Nitpick: can you use the same naming convention for the two FPD multiple bidder config tests?
Either make them:
- `exchange/exchangetest/firstpartydata-multiple-bidder-configs-valid.json`
- `exchange/exchangetest/firstpartydata-multiple-bidder-configs-invalid.json`
Or:
- `exchange/exchangetest/firstpartydata-multibidder-config-valid.json`
- `exchange/exchangetest/firstpartydata-multibidder-config-invalid.json`